src/main/java/net/shipilev/ForkJoin.java

```diff
@@ -16,9 +16,9 @@
 import java.util.concurrent.TimeUnit;
 
 @State(Scope.Benchmark)
-@Warmup(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
-@Measurement(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
-@Fork(5)
+@Warmup(iterations = 3, time = 1, timeUnit = TimeUnit.MILLISECONDS)
```

Comment:
Hi Alexey!

I have tried to print result of each benchmark before return with:
```System.out.println(acc + " ");```
And found that `@Warmup(iterations = 3)` and `@Measurement(iterations = 5)` produces more iteration than specified if their completion time less than 1 second.
Then after setting `time = 1, timeUnit = TimeUnit.MILLISECONDS` I got interesting results:
```
# Benchmark: net.shipilev.Disruptor.run

# Run progress: 0.00% complete, ETA 00:00:00
# Fork: 1 of 1
# Warmup Iteration   1: 384.449 ms/op
# Warmup Iteration   2: 435.573 ms/op
# Warmup Iteration   3: 433.340 ms/op
Iteration   1: 454.412 ms/op
Iteration   2: 440.380 ms/op
Iteration   3: 438.848 ms/op
Iteration   4: 436.011 ms/op
Iteration   5: 432.212 ms/op
...

# Benchmark: net.shipilev.SingleThread.run

# Run progress: 71.43% complete, ETA 00:00:07
# Fork: 1 of 1
# Warmup Iteration   1: 812.863 ms/op
# Warmup Iteration   2: 815.574 ms/op
# Warmup Iteration   3: 822.516 ms/op
Iteration   1: 815.691 ms/op
Iteration   2: 815.596 ms/op
Iteration   3: 815.986 ms/op
Iteration   4: 815.021 ms/op
Iteration   5: 829.840 ms/op

...

# Benchmark: net.shipilev.Streams.run

# Run progress: 85.71% complete, ETA 00:00:04
# Fork: 1 of 1
# Warmup Iteration   1: 555.185 ms/op
# Warmup Iteration   2: 340.118 ms/op
# Warmup Iteration   3: 338.866 ms/op
Iteration   1: 339.177 ms/op
Iteration   2: 341.390 ms/op
Iteration   3: 339.420 ms/op
Iteration   4: 339.130 ms/op
Iteration   5: 339.924 ms/op
```

For all benchmarks from this project 1 warmup iteration is enough and for some of them (Disruptor and SingleThread) best results are in 1st warmup iteration.